conductr_cli/sandbox_run_jvm.py

```diff
@@ -269,6 +270,19 @@ def validate_64bit_support():
         raise SandboxUnsupportedOsArchError()
 
 
+def validate_bintray_credentials(image_version, offline_mode):
+    """
+    Validates if the necessary Bintray credentials exists to start the ConductR cluster.
+    Credentials are necessary prior to ConductR 2.1.0. From onwards 2.1.0, no credentials are needed.
+    :param image_version: the ConductR image version
+    :param offline_mode: the offline mode flag
+    """
+    if not offline_mode:
+        major_version, minor_version, _ = version_parts(image_version)
+        if major_version == 2 and minor_version == 0:
```

Comment:
The conductr version validation logic will be replaced by another PR. In there, we will put all ConductR version validation into a separate python file. This file will contain functions to query if certain capabilities based on the ConductR version exist. Influenced by https://en.wikipedia.org/wiki/Gestalt_(Mac_OS)